pyat/developers.rst

```diff
@@ -58,17 +73,17 @@ recompiled.  To force recompilation, remove the build directory:
 Any changes to .py files are automatically reinstalled in the build, but to
 ensure any changes to .c files are reinstalled rerun:
 
-* ``python setup.py develop``
+* ``pip install -e .``
 
-If you get strange behaviour even after running setup.py develop again, then
+If you get strange behaviour even after running pip install develop again, then
 running the following, inside pyat, should fix it:
 
 * ``rm -rf build``
 * ``find at -name "*.pyc" -exec rm '{}' \;``
 * ``find at -name "*.so" -exec rm '{}' \;``
-* ``python setup.py develop``
+* ``pip install -e .``
```

Comment:
Is it possible to explain why I'm only able to install this module in editable mode during development?

It's not an obvious constraint. I'm fairly certain it was explained to me before, but I have already forgotten.

It's clearly attempting to compile the code (that's why I get a compiler error unless I add the `--std=c99` compiler flag), yet it still fails to properly install the module.

Reply:
Installing in editable mode allows changing python code and just restarting python without needing to run pip again. Otherwise you need to run `pip install` again after each modification. Of course, when modifying C code, or at 1st installation, you have to compile again.

Reply:
At least for me, it simply doesn't work outside of editable mode.

There's an error about missing modules, which I believe are compiled from C.

Reply:
`at.tracking.atpass` is missing, for example

Reply:
Ah, it's a different problem. Outside of editable mode, you have to work from a working directory different from pyat. The "current directory" is in your python path before "site-packages", so python tries to import from the local "at" instead of the standard "site-packages", and does not find the compiled modules. Try cd somewhere else, you'll see the installation is OK.

Reply:
Okay, good point